### ISSUE TITLE
Reorder data in a tile using the value of a property

### ISSUE THREAD

**Issue by iandees** [mapbox/tippecanoe#853]
I'm building a tileset with POI data that has an "importance" number on it. In areas where the data is very densely packed, I'd like to pre-collide the data out of the tile using the existing [`--gamma`](https://github.com/mapbox/tippecanoe#dropping-tightly-overlapping-features) flag, but I want to drop the "least important" places in a tile rather than whatever order tippecanoe would pick, so I propose that tippecanoe add a new [reordering](https://github.com/mapbox/tippecanoe#reordering-features-within-each-tile) flag that sorts features in a tile using the value of a given property name.

**Comment by ericfischer** [comment]
You ended up solving this with MTS right?

**Comment by iandees** [comment]
Yep. It'd still be nice to solve it in Tippecanoe, but I understand if it's not a priority for you.
